src/assets/i18n/nl.json

```diff
@@ -1,4138 +1,4187 @@
-  "A single bandwidth limit or bandwidth limit schedule in rclone format. Separate entries by pressing <code>Enter</code>. Example:  <samp>08:00,512 12:00,10MB 13:00,512 18:00,30MB 23:00,off</samp>. Units can be specified with the beginning letter: <samp>b</samp>, <samp>k</samp> (default), <samp>M</samp>, or <samp>G</samp>. See <a href=\"https://rclone.org/docs/#bwlimit-bandwidth-spec\" target=\"_blank\">rclone --bwlimit</a>.": "A single bandwidth limit or bandwidth limit schedule in rclone format. Separate entries by pressing <code>Enter</code>. Example:  <samp>08:00,512 12:00,10MB 13:00,512 18:00,30MB 23:00,off</samp>. Units can be specified with the beginning letter: <samp>b</samp>, <samp>k</samp> (default), <samp>M</samp>, or <samp>G</samp>. See <a href=\"https://rclone.org/docs/\\#bwlimit-bandwidth-spec\" target=\"_blank\">rclone --bwlimit</a>.",
-  "Additional      <a href=\"https://rsync.samba.org/ftp/rsync/rsync.html\" target=\"_blank\">rsync(1)</a>      options to include. Separate entries by pressing <code>Enter</code>.<br>      Note: The \"*\" character must be escaped with a backslash (\\*.txt) or used      inside single quotes ('*.txt').": "Additional      <a href=\"https://rsync.samba.org/ftp/rsync/rsync.html\" target=\"_blank\">rsync(1)</a>      options to include. Separate entries by pressing <code>Enter</code>.<br>      Note: The \"*\" character must be escaped with a backslash (\\\\*.txt) or used      inside single quotes ('*.txt').",
-  "Auth Token from alternate authentication - optional <a href=\"https://rclone.org/swift/#standard-options\" target=\"_blank\">(rclone documentation)</a>.": "Auth Token from alternate authentication - optional <a href=\"https://rclone.org/swift/\\#standard-options\" target=\"_blank\">(rclone documentation)</a>.",
-  "AuthVersion - optional - set to (1,2,3) if your auth URL has no version <a href=\"https://rclone.org/swift/#standard-options\" target=\"_blank\">(rclone documentation)</a>.": "AuthVersion - optional - set to (1,2,3) if your auth URL has no version <a href=\"https://rclone.org/swift/\\#standard-options\" target=\"_blank\">(rclone documentation)</a>.",
-  "Authentication URL for the server. This is the OS_AUTH_URL from an <a href=\"https://rclone.org/swift/#configuration-from-an-openstack-credentials-file\" target=\"_blank\">OpenStack credentials file</a>.": "Authentication URL for the server. This is the OS_AUTH_URL from an <a href=\"https://rclone.org/swift/\\#configuration-from-an-openstack-credentials-file\" target=\"_blank\">OpenStack credentials file</a>.",
-  "Change the default password to improve system security. The new password cannot contain a space or <b>#</b>.": "Change the default password to impro
```

Comment:
`{maxVCPUs}` is a placeholder here and should not be changed to `{maxVCPUs's}`

Reply:
changed as I did with the other ones': 'doel' is changed to 'target